src/main/java/frc/robot/commands/SetLEDOverride.java

```diff
@@ -0,0 +1,40 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.oi.IOperatorOI.OILED;
+import frc.robot.oi.IOperatorOI.OILEDState;
+import frc.robot.util.UpdateLEDInterface;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/latest/docs/software/commandbased/convenience-features.html
+public class SetLEDOverride extends InstantCommand {
```

Comment:
Why do we need this command? In RobotContainer we can do
`button.whenActive(() -> operatorOI.updateLED(OILED, OILEDState))`

Reply:
When we tried that, it didn't seem to run when disabled. Is there an alternate solution?